test/controllers/update/confirm.overseas.entity.details.controller.spec.ts

```diff
@@ -69,6 +75,20 @@ describe("Confirm company data", () => {
       expect(resp.text).toContain("January");
       expect(resp.text).toContain(testEntityNumber);
       expect(resp.text).toContain(testIncorporationCountry);
+      expect(resp.text).toContain(UPDATE_BANNER);
+    });
+
+    test(`renders the ${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL} page for the remove journey`, async () => {
+
+      mockGetApplicationData.mockReturnValueOnce(entityProfileModelMock).mockReturnValueOnce(entityProfileModelMock);
+      const resp = await request(app).get(`${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(BACK_LINK_FOR_UPDATE_OE_CONFIRM);
```

Comment:
The assertion on this line is passing but it needs to be more specific. Something like:
   `expect(resp.text).toContain(`${BACK_LINK_FOR_UPDATE_OE_CONFIRM}${config.JOURNEY_REMOVE_QUERY_PARAM}`);`

As query param should be on end of the back link. In fact, test may fail after changing this, until you incorporate the fix that @markpit highlighted above.

Reply:
Add specifics.